src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,30 @@
+package duke.main;
+
+import duke.utils.Constants;
+
+public class Duke {
+
+    private Ui ui;
+    private DukeWorker worker;
+
+    public Duke(String storagePath) {
+        ui = new Ui();
+        worker = new DukeWorker(storagePath);
+    }
+
+    public void run() {
+        worker.initializeWorker();
+        ui.welcomeMessage();
+        String request = "";
+        //while (!request.split(" ")[0].toLowerCase().equals("bye")) {
```

Comment:
Please delete unused commented-out code, or at least leave one blank line before and after it.